R/reportFE.R

```diff
@@ -1163,10 +1176,91 @@ reportFE <- function(gdx,regionSubsetList=NULL,t=c(seq(2005,2060,5),seq(2070,211
   )
   
   
+  ### temporary (!) industry non-energy use reporting
+  # note: only for REMIND-EU SSP2
+  
+  if (module2realisation["industry",2]=="fixed_shares") {
+    
+    if ("DEU" %in% getRegions(vm_prodFe)) {
+      
+      # some initializations required for building library with dplyr operations below
+      encar <- data <- value <- value_subsectors <- SSP <- Value_NonEn <- encar <- region <- period <- NULL
+      
+      
+      
+      # read in FE industry non-energy use trajectories from industry subsectors run
+      df.fe_nechem <- read.csv("C:/work/scripts/pik-piam/misc/pm_fe_nechem.cs4r", 
```

Comment:
Here's how you can access a data file from inside a package: Put the file in `inst/extdata` and access it with the command `system.file`. An example is found [here](https://github.com/pik-piam/mip/blob/605ccbb41e425f787916210fc959f058e8ddf49e/R/plotstyle.R#L43) for the file found [here](https://github.com/pik-piam/mip/blob/master/inst/extdata/plotstyle.csv)